src/main/java/com/unityTest/testrunner/service/CodeService.java

```diff
@@ -171,8 +174,54 @@ private void writeTestCaseToSuiteFile(PLanguage lang, File testFile, SuiteFile s
 		// Write suite file content
 		bos.write(suiteFile.getContent());
 		// Write test case content to file depending on language
-		// TODO FIX THIS
-		bos.write(String.format("\n%s", Utils.indent(caze.getCode(), 1)).getBytes());
+		switch (lang) {
+			case JAVA:
+				// TODO;
+				break;
+			case HASKELL:
+				// TODO: Consider putting code in file and replacing function name and body into read string

```

Comment:
I think this is necessary at this point...

Reply:
> I am assuming everything is good now? If so, could you approve the PR.

Almost!

Reply:
The suite template, should I save it with the Dockerfile and the entry script? Or should I create a new directory for it? If so, lmk where do want it.

And I think we should save the path in the application.yml file under `docker.build.haskell.suite-template`. Lmk if this sounds good or where do want it before I push the changes.

Reply:
That's perfect!